test/jasmine/tests/shapes_test.js

```diff
@@ -489,7 +489,7 @@ describe('Test shapes', function() {
             title: 'linked to date and category axes',
             xaxis: {
                 type: 'date',
-                range: ['2000-01-01', (new Date(2000, 1, 2)).getTime()]
```

Comment:
this old format for date axis range was mucking up the test code but not affecting the code being tested (because that's all after supplyDefaults cleans these) so it was useless.